src/test/java/com/clouway/cuse/SearchEngineContractTest.java

```diff
@@ -1001,6 +1035,15 @@ private void store(Employee... employees) {
     }
   }
 
+  private void storeAllObjects(List<Object> objects) {
+    long id = 1L;
+    for (Object object : objects) {
+      repository.store(id, object);
+      id++;
+    }
+    searchEngine.registerAll(objects);
```

Comment:
even better - additional usages with `List<User>` or `List<Ticket>` - to demonstrate that the method can be used with different types of objects.